.pre-commit-config.yaml

```
@@ -23,11 +24,11 @@ repos:
     -   id: debug-statements
 
 -   repo: https://github.com/pycqa/flake8.git
-    rev: 3.9.2
+    rev: 6.0.0
```

Let's go back to 3.9.2 and try again:
```suggestion
    rev: 3.9.2
```